common/model-views.cpp

```diff
@@ -720,9 +764,42 @@ namespace rs2
                         else
                         {
                             float tmp_value = value;
-                            if (ImGui::SliderFloat(id.c_str(), &tmp_value,
-                                range.min, range.max, "%.4f"))
+                            float temp_value_displayed = tmp_value;
+                            float min_range_displayed = range.min;
+                            float max_range_displayed = range.max;
+
+                            // computing the number of decimal dggits in the range's step of the option
```

Comment:
digits ... taken from the step options' property

Reply:
Done